DOCS/license-inclusion-principles.md

```diff
@@ -7,42 +7,41 @@ Determining whether a candidate license should be included on the SPDX License L
 * The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.
 * All OSI-approved licenses will be included on the SPDX License List.
   * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also at that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
-* Binary-only software licenses or licenses that do not provide for the availability of the source code will not be included on the SPDX License List.
+* Software licenses that apply only to executables and do not provide for the availability of the source code will not be included on the SPDX License List.
+* The license has identifiable and stable text; it is not in the midst of drafting. The license steward is committed to not modifying after addition to the list and to versioning new versions in the future.
   
 **Other factors (roughly in order of importance):**
-* The license substantially complies with one of the following open source definitions (even if not submitted for approval, as in the case of OSI) or these organization consider the license to meet their definition:
+1. The license substantially complies with one of the following open source definitions (even if not submitted for approval or these organization have not considered the license):
```

Comment:
It's good to have a list of these definitions, but I'm still against license submission by proxy. The "substantial" compliance perhaps conveys that SPDX discussion is no substitute for submission to other orgs. But it's not particularly clear how we'll go about discussing "substantially", either.

Kind of reminds me of federal courts deciding whether there's been a "material" effect under state contract law. Awkward.

To make this more concrete: Does this mean no more noncommercial licenses, from Creative Commons or anyone else?

Reply:
@kemitchell - you are a lawyer, you are used to awkward, gray area delineators!! :)
we do *not* want to be in the position of saying what does or does not meet someone else's definition (nor have we ever wanted to be in that position). That would be inappropriate.  So, we need that flexibility.  

Also, given all we've observed over the past year and a half, flexibility will allow the guidelines to flex as needed going forward (we don't want to have to update these too often, I'd hope!)

As for NC licenses - see historical background ;)  (which is why I wanted to include the historical background, so we don't have to keep answering that question and can say - here, read this!) I don't know - if it meets all the other criteria, I suppose it could be possible?  

case-by-case scenario... totality of the circumstances...

Reply:
+1 to @jlovejoy's comment here. 

Flexibility is relevant, and I think it should be understood in light of both (a) the remaining factors on this list; and (b) the intent of making the current guidelines changes, which is largely to make it permissible for more source-available licenses to be added to the list.